src/Angor/Avalonia/AngorApp/Composition/ServiceCollectionExtensions.cs

```diff
@@ -0,0 +1,94 @@
+using Angor.UI.Model;
+using Angor.UI.Model.Implementation;
+using Angor.UI.Model.Implementation.Projects;
+using AngorApp.Core;
+using AngorApp.Design;
+using AngorApp.Sections;
+using AngorApp.Sections.Browse;
+using AngorApp.Sections.Founder;
+using AngorApp.Sections.Home;
+using AngorApp.Sections.Portfolio;
+using AngorApp.Sections.Shell;
+using AngorApp.Sections.Wallet;
+using AngorApp.UI.Services;
+using Avalonia.Controls.Notifications;
+using Microsoft.Extensions.DependencyInjection;
+using Zafiro.Avalonia.Controls.Navigation;
+using Zafiro.Avalonia.Dialogs;
+using Zafiro.Avalonia.Services;
+using Zafiro.UI;
+
+namespace AngorApp.Composition;
+
+public static class ServiceCollectionExtensions
+{
+    public static IServiceCollection AddUIServices(this IServiceCollection services, Control parent)
+    {
+        var topLevel = TopLevel.GetTopLevel(parent);
+
+        return services
+            .AddSingleton<ILauncherService>(_ => new LauncherService(topLevel!.Launcher))
+            .AddSingleton<IDialog, DesktopDialog>()
+            .AddSingleton<IActiveWallet, ActiveWallet>()
+            .AddSingleton<INotificationService>(_ => new NotificationService(
+                new WindowNotificationManager(topLevel)
+                {
+                    Position = NotificationPosition.BottomRight
+                }
+            ))
+            .AddSingleton<UIServices>(sp => new UIServices(
```

Comment:
Wouldn't all those services just get resolved automatically? why this hack?

Reply:
You're absolutely right. Leftover from previous code... Will fix!